site/content/docs/main/deploy-options.md

```diff
@@ -7,16 +7,29 @@ If you don't have a cluster with that capability see the [Running without a Kube
 
 ## Installation
 
-### Recommended installation details
+Contour requires a secret containing TLS certificates that are used to secure the gRPC communication between Contour<>Envoy.
+This secret can be auto-generated by the Contour `certgen` job or provided by an administrator.
+Traffic must be forwarded to Envoy, typically via a Service of `type: LoadBalancer`.
+All other requirements such as RBAC permissions, configuration details, are provided or have good defaults for most installations.
 
-The recommended installation is for Contour to run as a Deployment and Envoy to run as a Daemonset. A secret containing
-TLS certificates should be used to secure the gRPC communication between them. A Service of `type: LoadBalancer` should
-also be created to forward traffic to the Envoy instances. The [example manifest][2] or [Contour Operator][12] will
-create an installation based on these recommendations.
+_Note: Contour Operator is alpha and therefore follows the Contour [deprecation policy][13]._
```

Comment:
this line should move down to below L23 since that's where the Operator is referenced.